src/app/components/CopyToClipboard/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC, useCallback, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Tooltip from '@mui/material/Tooltip'
+import ContentCopyIcon from '@mui/icons-material/ContentCopy'
+import IconButton from '@mui/material/IconButton'
+
+const clipboardTooltipDuration = 2000
+
+type CopyToClipboardProps = {
+  value: string
+}
+
+export const CopyToClipboard: FC<CopyToClipboardProps> = ({ value }) => {
+  const { t } = useTranslation()
+  let timeout: number | undefined
+  const ariaLabel = t('clipboard.label')
+  const [isCopied, setIsCopied] = useState(false)
+  const handleCopyToClipboard = useCallback(() => {
+    if (isCopied) {
+      return
+    }
+    window.navigator.clipboard.writeText(value)
+    setIsCopied(true)
+  }, [value])
+  const hideTooltip = useCallback(
+    () => (timeout = window.setTimeout(() => setIsCopied(false), clipboardTooltipDuration)),
+    [],
+  )
+
+  useEffect(() => {
+    return () => {
+      if (timeout) {
+        clearTimeout(timeout)
```

Comment:
Would be nice to have `window.clearTimeout` instead of `clearTimeout` for the sake of consistency.